stylesheets/_get-started.scss

```diff
@@ -256,3 +256,41 @@ ul.toc {
 
 #new-rating, #your-rating, #avg-rating {display:none;}
 #rating-section {padding-top:1em;}
+
+.bpm-suite{
```

Comment:
I think a lot of these styling improvements apply across the site, so we should fix it globally, rather than for just one page. For example the padding after a list is definitely something that I've noticed elsewhere.